obs-studio-server/source/nodeobs_service.cpp

```diff
@@ -685,7 +685,7 @@ static void ensure_directory_exists(std::string& path)
 
 	std::string directory = path.substr(0, last);
 
-	if (std::experimental::filesystem::is_directory(directory))
+	if (std::filesystem::is_directory(directory))
```

Comment:
isnt it should create dirs if is_direcotory is false?

Reply:
Correct, fixing, this must have been broken for a long time. It seems like the behavior was the same with the old API.